.github/workflows/release.yml

```diff
@@ -3,15 +3,18 @@ name: Release
 on:
   release:
     types: [published]
+  push:
```

Comment:
Could we put it into separate workflow?

Reply:
It depends on what we want to do after release. If we want to publish only the release version then it can be put into a separate workflow. 
If we want to publish release and then pre-release then I'm afraid that race condition may occur - release bumps minor version and commits it, pre-release bumps patch version - there is need for synchronization between them. Doing it via `need` was the easiest way I found to do it